docs/09-bootstrapping-kubernetes-workers.md

```diff
@@ -61,9 +61,9 @@ Install the worker binaries:
   sudo mv runc.amd64 runc
   chmod +x kubectl kube-proxy kubelet runc runsc
   sudo mv kubectl kube-proxy kubelet runc runsc /usr/local/bin/
-  sudo tar -xvf crictl-v1.12.0-linux-amd64.tar.gz -C /usr/local/bin/
-  sudo tar -xvf cni-plugins-amd64-v0.6.0.tgz -C /opt/cni/bin/
-  sudo tar -xvf containerd-1.2.0-rc.0.linux-amd64.tar.gz -C /
+  sudo tar -xvf crictl-v1.14.0-linux-amd64.tar.gz -C /usr/local/bin/
+  sudo tar -xvf cni-plugins-linux-amd64-v0.8.0.tgz -C /opt/cni/bin/
+  sudo tar --strip-components=1 -xvf containerd-1.2.6.linux-amd64.tar.gz -C /bin/
```

Comment:
Good catch on this. I too chose to upgrade everything - even compiled latest runs in a separate Ubuntu Server 19.04 VM (fun...). 

Sometime after 16.04, /usr/bin and /usr/local/bin (etc.) were consolidated down to a symlink'd "bin -> usr/bin/" 

So, running the original command on anything newer than 16.04 (like.. 18.04, or in our case 19.04) actually clobbers the symlink "bin -> usr/bin/", leaving the worker node with just /bin/ - and the tar's contents. 

Upon restart of the worker node, or even session close, it will no longer be possible to log in via SSH. The server must be recovered via GRUB (mounting root fs, then moving the wonky /bin/ dir out of the way, and finally restoring the symlink "bin -> usr/bin/") 

I ran this new proposed command on my bare-metal worker nodes (Ubuntu Server 19.04) and confirm that it keeps the symlink intact and places the underlying files into the symlink'd directory (in this case, usr/bin/).